subcell_analysis/compression_analysis.py

```diff
@@ -19,3 +18,106 @@ def asymmetry(fibers_df):
         #print(diff[index])
     print(len(diff))
     #write 
+
+
+def get_axis_distances_and_projections(fiber_points):
+    """
+    Returns the distances of fiber points from the axis
```

Comment:
By axis, you mean the higher order axis the control points are wrapping around, not the fiber axis itself? Would be good to clarify that, maybe nice to also add a few bullets to the readme with vocab, a suggestion:
- Monomer axis: the axis defined by the ReaDDy monomers wrapping the fiber axis
- FIber axis: the fiber axis itself which is defined by the control points
- Higher order axis(?): there's potentially some higher order helix that the fiber itself is wrapping when compressed

Reply:
Here, I am defining the axis as the line joining the first and last control point. Any suggestions for the nomenclature? Perhaps `end_to_end_axis`? 

The functions in this PR use what we referred to as the "backbone" of the fiber, which are the control points for cytosim, and a derived metric for ReaDDy. I could rename `fiber_points` to `polymer_trace` if that makes more sense.

Reply:
I think `fiber_points` is a good term for this, I might add a sentence to the readme at some point, but fine for now